packages/fast-form-generator-react/src/utilities/form-input.colors.ts

```diff
@@ -0,0 +1,12 @@
+export const colors: any = {
+    background000: "#000",
+    background100: "#1A1A1A",
+    background200: "#1E1E1E",
+    background300: "#212121",
+    background800: "#303030",
+    foreground000: "#FFF",
+    foreground200: "#F6F6F6",
+    foreground300: "#F2F2F2",
+    foreground800: "#D5D5D5",
+    pink: "#FB356D",
+};

```

Comment:
I would prefer that this is in the `form` folder and that this is for color constants so that the name of the file is `form.constants.style.ts` and the internals are:

```
export const background000: string: "#000";
...
```